.drone.yml

```diff
@@ -114,6 +114,21 @@ steps:
       kube_token:
         from_secret: dev_kube_token
 
+  - name: check dev up
+    pull: if-not-exists
+    image: alpine/curl:3.14
+    commands:
+      - chmod +x ./scripts/is-url-up.sh
```

Comment:
can't we just check in the file with the executable permission
`git update-index --chmod=+x <file>`

Reply:
Think this is one of those things where it was done in one repo then got copied everywhere else.

We mightt be able to go one step further looking @holly-blues's PR she has something cool that pulls in the script from a separate github 'space'. 
Maybe it would be possible to have that central script checked in with the ex permission as @stephenkaye-ho suggested. and we could just curl it here as Holly did..

Drone yml - line 130:
https://github.com/UKHomeOffice/callisto-ui/pull/63/files

Reply:
I've change it to match Holly's for now. Would changing the permissions (and maybe moving the location?) fall under a separate piece of work?

Reply:
As we know curl it as raw content the suggestion to change the file permission what make a difference. I agree that if we want to look at creating a specific common location for such scripts, it would fall under a separate piece of work